src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2219,8 +2224,20 @@ because xbuild doesn't support framework reference assemblies.
   </CreateItem>
 </Target>
 
+<Target Name="_PrepareNativeAssemblySources">
+  <ItemGroup>
+    <_BuiltAbis Include="$(_BuildTargetAbis)" />
```

Comment:
I think I see a problem here... 

`@(_BuildTargetAbis)` can include commas, as we have custom code to parse this here: https://github.com/xamarin/xamarin-android/blob/0d211ceb5c21f9f47d7894c1400596052055fb0c/src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs#L209

So if I take this example:
```xml
<Project>
  <PropertyGroup>
    <AbisComma>armeabi,armeabi-v7a,x86</AbisComma>
    <AbisSemiColon>armeabi;armeabi-v7a;x86</AbisSemiColon>
  </PropertyGroup>
  <ItemGroup>
    <SupportedAbisComma Include="$(AbisComma)" />
    <SupportedAbisSemiColon Include="$(AbisSemiColon)" />
  </ItemGroup>
  <Target Name="Build">
    <Message Text="Comma: $(AbisComma)" Importance="High" />
    <Message Text="Comma Batched: %(SupportedAbisComma.Identity)" Importance="High" />
    <Message Text="SemiColon: $(AbisSemiColon)" Importance="High" />
    <Message Text="SemiColon Batched: %(SupportedAbisSemiColon.Identity)" Importance="High" />
  </Target>
</Project>
```

It outputs:
```
  Comma: armeabi,armeabi-v7a,x86
  Comma Batched: armeabi,armeabi-v7a,x86
  SemiColon: armeabi;armeabi-v7a;x86
  SemiColon Batched: armeabi
  SemiColon Batched: armeabi-v7a
  SemiColon Batched: x86
```
The *Batched messages should be printing each one individually. But MSBuild doesn't support commas to delimit item groups...